tox.ini

```diff
@@ -14,3 +14,10 @@ commands =
 	{envpython} pycodestyle.py --statistics pycodestyle.py
 	{envpython} pycodestyle.py --doctest
 	{envpython} -m testsuite.test_all
+
+
+[testenv:flake8]
+deps =
+    flake8
+commands =
+    flake8 pycodestyle.py
```

Comment:
What's the rationale behind running flake8 on pycodestyle? Not specifically disagreeing, just seems to not be part of the "Switch Travis to use tox" issue.